web/src/stream_edit.ts

```diff
@@ -359,6 +368,16 @@ export function archive_stream(stream_id: number, $alert_element: JQuery): void
     });
 }
 
+export function unarchive_stream(stream_id: number, $alert_element: JQuery): void {
```

Comment:
Missed this before, but is this separate function required? `do_unarchive_stream` can directly have this code.